node/src/components/linear_chain.rs

```diff
@@ -270,10 +243,6 @@ impl<I> LinearChain<I> {
                 // TODO: disconnect from the sender.
                 continue;
             }
-            if known_signatures.proofs.contains_key(&fs.public_key) {
-                // Don't send finality signatures we already know of.
-                continue;
-            }
```

Comment:
Why did you remove this?

Reply:
It's duplicated: It's already in line 236.